### DISCUSSION TITLE
Excel Import with dropdown

### DISCUSSION THREAD

**Discussion by mgdm187** [serenity-is/Serenity#5609] [Help]
Hi,
I would like to create excel import. Before selecting the file that they want to import, I want to add a dropdown to select the type of import (e.g. Webinar, Meeting).

How can I add a dropdown in excel import?

Thank you!

**Comment by edwardch** [comment]
Check out @minhhungit's nice example on for dropdowns: [Create Serenity Toolbar Dropdown Button](https://minhhungit.github.io/2020/06/10/008-create-serenity-toolbar-dropdown-button/) and modify the code for imports...
